### PR TITLE
Coverage create next progression

### PR DESCRIPTION
**Pull request by chrisdugne**
## Detailed purpose of the PR

```
actions/progressions
├── create-chapter-progression.js
├── create-level-progression.js
├── create-next-progression.js
├── select-progression.js
└── synchronize.js
```

+ coverage for `create-next-progression`

### Testing strategy

- [x] I ensured my code is tested: 
- [x] I took care to mention breaking changes and extra libs if appropriate

### CODE REVIEW

File: src/redux/actions/progressions/synchronize.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+import type {StoreAction, ErrorAction} from '../../_types';
+import {getToken, getBrand} from '../../utils/state-extract';
+import {isDone} from '../../../utils/progressions';
+
+export const SYNCHRONIZE_REQUEST = '@@progression/SYNCHRONIZE_REQUEST';
+export const SYNCHRONIZE_SUCCESS = '@@progression/SYNCHRONIZE_SUCCESS';
+export const SYNCHRONIZE_FAILURE = '@@progression/SYNCHRONIZE_FAILURE';
+export const CREATE_NEXT_REQUEST = '@@progression/CREATE_NEXT_REQUEST';
+export const CREATE_NEXT_SUCCESS = '@@progression/CREATE_NEXT_SUCCESS';
+export const CREATE_NEXT_FAILURE = '@@progression/CREATE_NEXT_FAILURE';
```
Comment (VincentCATILLON) [inline src/redux/actions/progressions/synchronize.js]: remove these 3 constants please

File: src/redux/actions/progressions/create-level-progression.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+import {ENGINE} from '../../../const';
+import {getMostAccurateRef} from '../../../modules/reference';
+
+const ENGINE_VERSION = '1';
+const ENGINE_CONFIG_VERSION = '1';
+
+export const SYNCHRONIZE_REQUEST = '@@progression/SYNCHRONIZE_REQUEST';
+export const SYNCHRONIZE_SUCCESS = '@@progression/SYNCHRONIZE_SUCCESS';
+export const SYNCHRONIZE_FAILURE = '@@progression/SYNCHRONIZE_FAILURE';
```
Comment (VincentCATILLON) [inline src/redux/actions/progressions/create-level-progression.js]: remove these 3 constants please

File: src/redux/actions/progressions/create-level-progression.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+const ENGINE_CONFIG_VERSION = '1';
+
+export const createLevelProgression = (level: Level) => {
+  const ref = getMostAccurateRef(level);
+  const engine: Engine = {ref: ENGINE.LEARNER, version: ENGINE_VERSION};
+  // @todo use universalRef
+  const content: GenericContent = {type: CONTENT_TYPE.LEVEL, ref};
+  const engineConfig: EngineConfig = {
+    version: ENGINE_CONFIG_VERSION,
+    livesDisabled: level.infiniteLives
```
Comment (Fausto95) [inline src/redux/actions/progressions/create-level-progression.js]: c'est pas un bool ici?
Tu consideres 0 || >=1 c'est ça?

Comment (chrisdugne) [reply src/redux/actions/progressions/create-level-progression.js]: jai juste deplacé le code, jai rien rajouté de particulier;
`level.infiniteLives` est un boolean je pense

**Comment by chrisdugne** [comment]
[Remonter le coverage redux à 100%](https://trello.com/c/UBl8CyXB/1374-remonter-le-coverage-redux-%C3%A0-100)

**Pull request merged**